src/components/Article.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function Article(obj) {
```

Comment:
```suggestion
function Article({ title, date, text }) {
```
делай деструктуризацию props

Reply:
я так и делал, но линтер выдавал такую ошибку: 'title' is missing in props validationeslintreact/prop-types